src/hotspot/share/runtime/vmThread.hpp

```diff
@@ -161,10 +121,14 @@ class VMThread: public NamedThread {
   static void create();
   static void destroy();
 
+  static void until_executed(VM_Operation* op);
+
  private:
   // VM_Operation support
   static VM_Operation*     _cur_vm_operation;   // Current VM operation
-  static VMOperationQueue* _vm_queue;           // Queue (w/ policy) of VM operations
+  static VM_Operation*     _next_vm_operation;
```

Comment:
Comment to match `// Current VM operation` above?

Reply:
Fixing